packages/pharos/src/components/select/pharos-select.ts

```diff
@@ -45,7 +45,11 @@ export class PharosSelect extends ObserveChildrenMixin(FormMixin(FormElement)) {
   }
 
   protected firstUpdated(): void {
-    this._setOption();
+    if (!this.value) {
+      this._setOption();
+    } else {
+      this._select.value = this.value;
+    }
```

Comment:
would we want to check if `this.value` exists as a possible `option`? 
or is it kinda on the consumer at that point? 😅

Reply:
Hmm I don't think setting `value` on the select will do anything if it doesn't exist but I can check what React does.

Reply:
Added a check